packages/libs/src/services/solana/transactionHandler.ts

```diff
@@ -304,17 +312,27 @@ export class TransactionHandler {
     const startTime = Date.now()
     if (retry) {
       ;(async () => {
+        const connections =
+          this.fallbackConnections?.filter(
+            (conn) => conn.rpcEndpoint !== this.connection.rpcEndpoint
+          ) ?? []
+        connections.push(this.connection)
         let elapsed = Date.now() - startTime
+        let currConnIdx = 0
         // eslint-disable-next-line no-unmodified-loop-condition
         while (!done && elapsed < this.retryTimeoutMs) {
+          const conn = connections[currConnIdx % connections.length]
           try {
-            sendRawTransaction()
+            sendRawTransactionToConn(conn!)
           } catch (e) {
             logger.warn(
-              `transactionHandler: error in send loop: ${e} for txId ${txid}`
+              `transactionHandler: error in send loop: ${e} for txId ${txid} to ${
+                conn!.rpcEndpoint
+              }`
             )
           }
           sendCount++
```

Comment:
should be able to reuse sendCount for currConnIdx

Reply:
tru